layouts/partials/menus/sidebar.html

```diff
@@ -1,20 +1,22 @@
 {{ $input := . }}
 
 
-<menu class="sidebar-menu">
-    {{- $currentNode := $input.context }}
-    {{- range $input.menu }}
-        <li role="presentation" class="sidebar-menu__item">
-            <a
-                role="menuitem"
-                href="{{ .URL }}"
-                {{- if or ($currentNode.IsMenuCurrent .Menu .) ($currentNode.HasMenuCurrent .Menu .) }}
-                    class="sidebar-menu__link sidebar-menu__link--active"
-                {{- else }}
-                    class="sidebar-menu__link"
-                {{- end }}>
-                {{- or (T (printf "menu.%s" .Identifier)) .Name }}
-            </a>
-        </li>
-    {{- end }}
-</menu>
+<nav role="navigation">
```

Comment:
Why this change? Why no menu anymore? The `baseof` template already wraps the menu and everything else of the sidebar into a `nav` element.

Reply:
I didn't check the usage. Thanks. The `nav` could be removed from here at all. But I would suggest using `ul` instead of `menu` - [docs](https://www.w3.org/WAI/tutorials/menus/structure/#unordered-list).

Reply:
Here is the warning about the menu:
![image](https://github.com/user-attachments/assets/32287c2b-fbdd-49e2-9fb2-a5b892be6b70)

Reply:
Let me try to sum up my findings. If we use `<ul>` or `<menu>` makes absolutely no difference.

> The `<menu>` element is described in the HTML specification as a semantic alternative to `<ul>`,
> but treated by browsers (and exposed through the accessibility tree) as no different than `<ul>`.
> -- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu

So we can continue using `<menu>` here to clarify (under developers) that this is not just another unordered list. The actual problem here, that lead to the [error](https://dequeuniversity.com/rules/axe/4.6/aria-required-parent), was that we used `role="menuitem"` but forgot the `role="menu"` here on the unordered list. However, the roles `menu` and `menuitem` seem not necessary (but also not wrong?) when used inside a `<nav>` element.

Finally, what's written about the [`navigation` role on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role) is that each navigation should have a label (e.g., `Main`, `Secondary`, `Footer`). And since we have two navigation elements that are actually the same (header and sidebar), they should have the same label (`Main`). But also: "`<nav>` is intended only for a major block of navigation links; typically the `<footer>` element often has a list of links that don't need to be in a `<nav>` element." ([MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#attributes)). So I guess we should remove the `<nav>` element from the footer and secondary menu.

Reply:
To stay consistent, I would keep the `<nav>` elements as is. We just should label them. In our four menu partials, I would stay with `<menu>` for the given reasons. If you change it to `<ul>`, please change all of them.

Reply:
This issue was discussed during our recent call. We agreed to resolve this issue by following [mdn web docs](https://developer.mozilla.org/en-US/) and [W3C WAI](https://www.w3.org/WAI/tutorials/menus/)